de.dlr.sc.virsat.docs.feature/src/docs/VirSat_Core_Server_Manual.adoc

```diff
@@ -0,0 +1,234 @@
+= Virtual Satellite 4 Core: Server Manual
+:imagesdir: images
+:title-logo-image: images/title/VirtualSatellite_Server.jpg
+:toc:
+:toclevels: 3
+:experimental: 
+
+== Introduction
+
+Virtual Satellite can be run headless as a server that provides a REST API. 
+The REST API can be used to access and manipulate the Virtual Satellite model without the user interface.
```

Comment:
I think this paragraph needs to be expanded a bit. I don't want us to explain in detail what REST is, but somebody who doesnt know what REST is (or just quickly googled it) shiould be able to read this. As a first step we could improve this by mentioning here that the API can be used to 'acess and manipulate the Virtual Satellite model _remotely_ and without the _desktop_ interface". I think that would make the purpose a bit clearer. But there's probably some more than can be done.

Reply:
Sure, I tried to rewrite it in a more user friendly way